docs/src/components/header/ThemeModeToggle.tsx

```diff
@@ -51,6 +52,7 @@ export default function ThemeModeToggle() {
       <Tooltip title={calculatedMode === 'dark' ? 'Turn on the light' : 'Turn off the light'}>
         <IconButton
           color="primary"
+          size="small"
```

Comment:
If we change the hydrated version, we also need to change the placeholder for hydration. Right now, in https://next--material-ui.netlify.app/material-ui/ we have:

https://github.com/mui/material-ui/assets/3165635/016f50bc-555c-4bea-bb80-8846fb6e787b

Fixed with a small commit straight to HEAD: 3c0b821e0f65aac3a8aba6584b52c153c3951611.

After: 

https://github.com/mui/material-ui/assets/3165635/72feb62f-c3dd-4069-8220-07421dbecf30

(I noticed this while I was navigating the docs on the go)